back/src/domains/establishment/use-cases/SendBeneficiariesPdfAssessmentsEmails.ts

```diff
@@ -19,42 +18,42 @@ type SendBeneficiaryAssessmentEmailsOutput = {
 };
 
 export class SendBeneficiariesPdfAssessmentsEmails extends TransactionalUseCase<
-  void,
+  {
+    conventionEndDate: DateRange;
+  },
   SendBeneficiaryAssessmentEmailsOutput
 > {
-  protected inputSchema = z.void();
+  protected inputSchema = z.object({
+    conventionEndDate: z.object({
+      from: z.date(),
+      to: z.date(),
+    }),
```

Comment:
Quittes à faire un schema avec ça, tu penses quoi de rajouter un refine qui s'assure que `from` est bien avant `to`. Pour éviter les bugs.